cluster/node-pools/master-default/userdata.yaml

```diff
@@ -864,3 +864,30 @@ write_files:
           qps: {{ .Cluster.ConfigItems.event_rate_limit_config_qps }}
           burst: {{ .Cluster.ConfigItems.event_rate_limit_config_burst }}
 {{ end }}
+
+  - owner: root:root
+    path: /etc/kube-flannel/net-conf.json
+    content: |
+      {
+        "Network": "10.2.0.0/15",
+        "Backend": {
+          "Type": "vxlan"
+        }
+      }
```

Comment:
I wonder if we want to leave this config in the configMap in order to make flannel more self-containd. This is a [flag to flannel](https://github.com/flannel-io/flannel/blob/master/main.go#L131) itself and is not needed by kubelet (As far as I can tell). So from that perspective it can stay with flannel.

Reply:
Fixed in the next commit. Thanks!